README.md

```diff
@@ -30,3 +30,23 @@ _Optional_ The number of days after which unreleased commits should be considere
 
 ### `commit-messages-lines: 1`
 _Optional_ Limit the number of first x lines from commit messages that will be added in the issue description. No truncation when set to `0`. Default is `1`.
+
+
+## Auto close issue after release
+To automatically close the "pending release" issue generated by the action after the release is successful, you can add a workflow trigger that runs the action after the release is published.
+
+```yaml
+name: notify-release
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  schedule:
+    - cron: '30 8 * * *'
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify release
+        uses: nearform/github-action-notify-release@v1
+```
```

Comment:
let's not add this, let's just replace the basic example with this, which is the default way it should be used

Reply:
I have updated the README to make it a default option